proto/credentials.proto

```diff
@@ -10,6 +10,8 @@ message SSHAuth {}
 
 message CloudEnvironment {}
 
+message ADC{}
```

Comment:
By default GCP SDKs will attempt to load credentials from ADC and from the metadata server (available in GCP runtime environments).  `CloudEnvironment` is used for S3 for the same.

Reply:
ah, that makes sense. I'll use that message instead. Thanks.